### DISCUSSION TITLE
Scammed by our developer

### DISCUSSION THREAD

**Discussion by Hu99icaneStudios** [community/community#130304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I notify Github and Vercel communities of a scum-of-the-earth devloper who has stolen all the funds from our project. The options to report on github are not for criminal activity. His details as follow:

- https://github.com/sunshine0709
- Vercel - topdev0104
- Discord - partner6844 | User ID #880275150786945064
- Telegram - @partner0418

**Comment by V4nshaj** [comment]
I'm sorry to hear about your situation. If you believe someone has committed criminal activity, such as stealing funds from your project, here are steps you can take to report this:

1. Contact Law Enforcement:
Start by reporting the incident to your local law enforcement agency or the relevant authorities where the developer resides. Theft and fraud are serious offenses that should be handled by legal authorities.
2. GitHub Reporting:
While GitHub's reporting options are typically for code and repository-related issues rather than criminal activities, you can still report the user for violating GitHub's terms of service. This may prompt GitHub to investigate and possibly suspend the user's account if they find evidence of wrongdoing. To report on GitHub:
Go to the user's profile (https://github.com/sunshine0709).
Click on the "···" button next to the user's name.
Select "Report user" and provide details of the issue.
3. Vercel Reporting:
Similarly, Vercel may have terms of service violations related to fraudulent activities. You can report the user by:
Contacting Vercel support directly through their official support channels.
Providing details such as the user's profile (topdev0104) and explaining the situation.
4. Discord and Telegram Reporting:
Discord and Telegram have reporting mechanisms for users engaging in illegal activities. You can report the user by:
Discord: Visit their support page or send an email to abuse@discord.com with the user's ID (880275150786945064) and details of the incident.
Telegram: Use Telegram's abuse reporting tool within the app or contact their support team with the user's username (@partner0418) and details of the issue.

IF MY ANSWER HELPED YOU PLEASE MARK MY ANSWER

**Reply by ebndev** [reply]
Hi @V4nshaj, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ebndev** [comment]
I’m sorry that you’re having to go through this, @Hu99icaneStudios. 

As this is a personal matter within your organization, we are unable to assist with your issue.

Here's all of the info with our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) if you feel this will be appropriate for your case.

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.
